apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/api/Range.java

```diff
@@ -28,39 +32,45 @@
     void validate();
 
     /**
-     * Returns the min value - the left boundary of the range
+     * Returns the min value of the range - the bottom boundary of the range
      *
      * @return the left boundary of the range
      */
     T from();
 
+    default T bottom() {
```

Comment:
I there a reason to have all of these methods: bottom(), from(), min() ?

Reply:
removed bottom and min methods;